exercises/error-handling/src/main/java/ErrorHandling.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Optional;
+
+class ErrorHandling {
+
+    public void handleErrorByThrowingException() throws Exception {
```

Comment:
Use of `public` in this file can also be changed to package private!

Also, I think we can remove `throws Exception` from these - it might give a little bit too much away to the users 😉 You should still be able to correctly throw `UnsupportedOperationException` even without this 🙂

Reply:
Ok 🙂